src/test/components/TrackContextMenu.test.js

```diff
@@ -250,6 +252,7 @@ describe('timeline/TrackContextMenu', function() {
 
       const isolateLocalTrackItem = () => getByText('Only show "DOM Worker"');
       const hideDOMWorker = () => getByText('Hide "DOM Worker"');
+      const hideAllTracksByType = () => getByText('Hide "DOM Worker"');
```

Comment:
This does not do what it's supposed to do.. You are still getting`Hide "DOM Worker"` item with this instead of `Hide all "DOM Worker" tracks`.
Also we don't use this function anywhere. We need to write a test for this item and see that it's working correctly.